### ISSUE TITLE
Configuration file reading

### ISSUE THREAD

**Issue by Xenanex** [dell/PyU4V#92]
Hello,

I'm working with this library for my company and I have a trouble with the configuration file:

In my code I have this:
```
import PyU4V

PyU4V.U4VConn.file_path = 'xxxxx/PyU4V.conf'

...

conn = PyU4V.U4VConn(server_ip=ip, array_id=serial)
```

But this simple code return me this:
```
Traceback (most recent call last):
File "./powermax.py", line 596, in <module>
conn = PyU4V.U4VConn(server_ip=ip, array_id=serial)

File "xxx/PyU4V/univmax_conn.py", line 95, in __init__ raise exception.MissingConfigurationException
PyU4V.utils.exception.MissingConfigurationException: PyU4V settings not be loaded, please check file location or univmax_conn input parameters.
```

After a day of research I find a solution:
`PyU4V.univmax_conn.file_path = 'xxxx/PyU4V.conf'`

I think it's because the variable has remove from the class to be place on the file or maybe the Documentation is not good.

Documentation link:
https://pyu4v.readthedocs.io/en/latest/configuration.html#alternative-pyu4v-conf-load-using-u4vconn-file-path

PS: I maybe did a mistake during the rewrite of the error.

**Comment by MichaelMcAleer** [comment]
Hi!

Thanks for highlighting this, I had a look at our docs and the code and you are correct, the docs should indeed read `PyU4V.univmax_conn.file_path = 'xxxx/PyU4V.conf'` because it is defined outside of the `U4VConn()` class. I'm so used to having my `PyU4V.conf` file in `~/.PyU4V` that this one went unnoticed. I will get it fixed in the next commit.

Thanks again!
Michael

**Comment by Xenanex** [comment]
No problem and thanks for your reply

**Comment by MichaelMcAleer** [comment]
Hi,

This issue has been resolved in pull request #94, once CI has been run on the changes it will be merged with master and reflected in our online documentation.

Thanks again for highlighting this issue,
Michael
